app/controls_test.go

```diff
@@ -47,7 +41,13 @@ func TestControl(t *testing.T) {
 		return xfer.Response{
 			Value: "foo",
 		}
-	}))
+	})
+	client, err := xfer.NewAppClient(probeConfig, ip+":"+port, ip+":"+port, controlHandler)
+	if err != nil {
+		t.Fatal(err)
+	}
+	client.ControlConnection()
```

Comment:
> Why not roll this into xfer.NewAppClient?

Not all users of appclient want to start a control connection.  Tests are one example, another is using app client to fetch the ID of the app, and then using that to decide if we should start a new control connection.